### DISCUSSION TITLE
Automatically use new record after creating within createOptionForm

### DISCUSSION THREAD

**Discussion by aaronmeder** [filamentphp/filament#8538] [Help] [answered]
Hello Filament community

It's fun so far to do my first project with a Filament admin panel 🥳

I have a Select field with a `createOptionForm` to create a new record in place.
That works great.

Is there an option to automatically use the newly created record in the select?

At the moment the record is available in the Select field after creation but needs to be manually selected:
![Screenshot 2023-09-16 at 13 16 33](https://github.com/filamentphp/filament/assets/5564163/9c9bb019-1173-47cb-a090-ad11d8e6fff2)

Thanks for help.
Aaron

**Comment by leandrocfe** [comment, marked as answer]
try this with `->searchable()->preload()`
are you using `->relationship()`?

**Reply by aaronmeder** [reply]
This works perfectly, thanks! Searchable was something I wanted to add anyway 💯

**Comment by e-m-s-y** [comment]
I'm using this with ->relationship() and unfortunatly the newly created record is not automatically selected. I've tried preload and searchable but it didn't make a difference.
